src/components/CodeEditor/PythonEditor.js

```diff
@@ -0,0 +1,154 @@
+import React, { useState, useEffect } from "react";
+
+import { usePython } from "./react-py";
+
+import CodeMirrorEditor from "./CodeMirror";
+
+import ArrowRightIcon from "@mui/icons-material/ArrowRight";
+import RestartAltIcon from "@mui/icons-material/RestartAlt";
+import CableIcon from "@mui/icons-material/Cable";
+
+import { Box, Button, Grid, Typography } from "@mui/material";
+
+const PythonEditor = ({
+  initialCodeEditorValue,
+  disableEditing,
+  disableRun,
+}) => {
+  let { runPython, stdout, stderr, isLoading, isRunning } = usePython();
+  const [pythonEditorCode, setPythonEditorCode] = useState(
+    initialCodeEditorValue
+  );
+  const [codeRan, setCodeRan] = useState();
+  let [codeExecuted, setCodeExecuted] = useState(stdout, stderr);
+
+  useEffect(() => {
+    if (stdout) {
+      setCodeExecuted(stdout);
+    } else {
+      setCodeExecuted(stderr);
+    }
+  }, [stdout, stderr]);
+  return (
+    <Box
+      className="PythonCodeEditor"
+      sx={{
+        my: 3,
+      }}
+    >
+      <Box
+        sx={{
+          p: "2px",
+          border: "1px solid #6D6D6D",
```

Comment:
Reference theme colors instead of hardcoding `#6D6D6D` in each place because these can change:
https://github.com/navgurukul/bhanwari-devi/blob/f0b79712ebe46fc4ade63f68fde662b36319b70a/src/theme/theme.js#L71

Reply:
Done. This color is currently under theme.palette.text.secondary, would this make the color volatile?

Reply:
@RTC-13 Well, the question is whether we want to have it always match the color of the text, which I'm assuming we do. We should ask Narendra about this to confirm.